core/src/components/range/range.ionic.scss

```diff
@@ -0,0 +1,147 @@
+@import "./range";
```

Comment:
We should split this file into `range.native` and `range.common`. The Accordion component is an example of this: 

- [accordion.common.scss](https://github.com/ionic-team/ionic-framework/blob/next/core/src/components/accordion/accordion.common.scss)
- [accordion.native.scss](https://github.com/ionic-team/ionic-framework/blob/next/core/src/components/accordion/accordion.native.scss)

The `range.common.scss` file should contain styles shared across all three themes and should be imported by both `range.native.scss` and `range.ionic.scss`. The `range.native.scss` file should contain styles specific to the `ios` and `md` themes. Styles using properties like `font-weight`, `padding`, etc. for `ios` and `md`, defined by the design tokens for the `ionic` theme, should be placed in `range.native.scss`.

Let me know if you need further clarification! 🙂

Reply:
I think I started doing what you asked with my latest commit but I unfortunately didn't understand what else to move to the new `range.native.css` file.

Reply:
I wasn't really sure what to do with values and variables used in `range.common.scss` that had no translation to our design tokens so I left them there. I can still move them to `range.native.scss` and write the values in `range.ionic.scss` manually if you want.